src/main/java/org/openstreetmap/atlas/checks/distributed/ShardedIntegrityChecksSparkJob.java

```diff
@@ -219,46 +204,27 @@ public void start(final CommandMap commandMap)
                             .collect(Collectors.toList());
 
                     // Set spark UI job title
-                    this.getContext().setJobGroup("0",
-                            String.format("Running checks on %s",
-                                    tasksForCountry.get(0).getCountry()));
-                    // Run each task in spark, producing UniqueCheckFlagContainers
-                    final JavaPairRDD<String, UniqueCheckFlagContainer> rdd = this.getContext()
-                            .parallelize(tasksForCountry, tasksForCountry.size())
+                    this.getContext().setLocalProperty("callSite.short", String
```

Comment:
What is `callSite.short` ?

Reply:
It is a special property that the Spark UI picks up for labeling. 
https://stackoverflow.com/questions/41903342/how-to-change-job-stage-description-in-web-ui